### PR TITLE
Fixes #8849 - installs foreman_gutterball

### PR DESCRIPTION
**Pull request by dustints**
* installs foreman_gutterball
* fixes devel installation with gutterball

### CODE REVIEW

**Comment by komidore64** [comment]
this looks sane to me!

**Comment by dustints** [comment]
removing submodules that got added accidentally.

File: Puppetfile.lock [hunk 1]
```diff
@@ -178,16 +178,6 @@ GIT
       puppetlabs-stdlib (>= 2.2.1)
 
 GIT
-  remote: https://github.com/puppetlabs/puppetlabs-postgresql
```
Comment (ehelms) [inline Puppetfile.lock]: Why is this getting removed here? I must be missing something.

Comment (dustints) [inline Puppetfile.lock]: didn't notice that! will restore back.
was trying to get librarian-puppet update going must have removed that while experimenting.

Comment (dustints) [inline Puppetfile.lock]: addressed

**Comment by dustints** [comment]
@ehelms addressed

**Comment by ehelms** [comment]
ACK

**Pull request merged**